ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -283,6 +283,32 @@ UtxoSignedTransaction receiveTransaction(
             @NotNull FlowSession session
     );
 
+    /**
+     * Sends the wire transaction to counterparty sessions.
+     *
+     * @param sessions The counterparties who receive the transaction.
+     * @param signedTransaction The {@link UtxoSignedTransaction} to send.
+     * @throws CordaRuntimeException If transaction verification fails on the receiving sessions.
```

Comment:
I do not think this applies to this case.

Reply:
Isn't this referring to this part of the flow?
`val sendingTransactionResult = it.receive(Payload::class.java)
            if (sendingTransactionResult is Payload.Failure) {
                throw CordaRuntimeException(
                    sendingTransactionResult.message
                )
            }`

Reply:
Yes. That propagates back the exceptions to the receiver side.
But Transaction verification cannot fail since it does not happen in the receive.